rclc_parameter/src/rclc_parameter/parameter_server.c

```diff
@@ -360,6 +360,8 @@ rcl_ret_t rclc_parameter_server_init_default(
     parameter_server, node, &DEFAULT_PARAMETER_SERVER_OPTIONS);
 }
 
+#define CHECK_BOOL_RET(ret) do {if (!(ret)) {return RCL_RET_ERROR;}} while (0)
```

Comment:
can be more flexible in future use?

```suggestion
#define CHECK_BOOL_RET(ret, error_type) do {if (!(ret)) {return error_type;}} while (0)
```